src/pages/FinalAccessReview.js

```diff
@@ -15,6 +16,7 @@ import { Config } from '../libs/config';
 import { Models } from '../libs/models';
 import { Storage } from '../libs/storage';
 import * as Utils from '../libs/utils';
+import TranslatedDULComponent from '../components/TranslatedDULComponent';
 
 
 class FinalAccessReview extends Component {
```

Comment:
Technical debt ... I don't think we can get to this component any more. The only reference to the path defined for this is in the chair console, but it's an unused method:
```
  openFinalAccessReview = (referenceId, electionId, rpElectionId) => (e) => {
    this.props.history.push(`${'final_access_review'}/${referenceId}/${electionId}`);
  };
```
Instead of updating this component, we should remove it. Feel free to spin this off into a new ticket to keep this PR more manageable.